pkg/base/linuxutils/fs/fs_helper.go

```diff
@@ -312,3 +319,20 @@ func (h *WrapFSImpl) GetFSType(device string) (string, error) {
 	}
 	return strings.TrimSpace(stdout), err
 }
+
+// GetFSUUID detect FS UUID from the provided device using lsblk --output UUID
+// Receives file path of the device as a string
+// Returns error if something went wrong
+func (h *WrapFSImpl) GetFSUUID(device string) (string, error) {
+	var (
+		cmd    = fmt.Sprintf(GetFsUUIDCmdTmpl, device)
+		stdout string
+		err    error
+	)
+	if stdout, _, err = h.e.RunCmd(cmd,
+		command.UseMetrics(true),
+		command.CmdName(fmt.Sprintf(GetFsUUIDCmdTmpl, ""))); err != nil {
```

Comment:
It returns empty string template `"lsblk %s --output UUID --noheadings"` ?

Reply:
Yes, for metrics selection in prometheus lib